tools/spancmp/src/main.rs

```diff
@@ -0,0 +1,289 @@
+//! helper to extract span stats from a chrome trace file
+//! spec: <https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#heading=h.puwqg050lyuy>
+
+use std::{
+    collections::HashMap,
+    fs::File,
+    io::{BufReader, Read},
+    ops::Div,
+};
+
+use clap::Parser;
+use regex::Regex;
+use serde::Deserialize;
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+/// A span from the trace
+#[derive(Deserialize, Debug)]
+struct Span {
+    /// name
+    name: String,
+    /// phase
+    ph: String,
+    /// timestamp
+    ts: f32,
```

Comment:
```suggestion
    name: String,
    phase: String,
    timestamp: f32,
```
and fix references. I see no reason to shorten the member names, nor to document with what the member names are/should be.

Reply:
The fields have the same name as in the json. They can be annotated with the serve annotation to rename them, but I tend to prefer to stick to the same name as the json being deserialised when not part of the public api